docs/getting-started/dcrd-and-dcrwallet-cli-arguments.md

```diff
@@ -0,0 +1,181 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Options2.svg" /> `dcrd` and `dcrwallet` CLI Arguments
+
+---
+Both the `dcrd` and `dcrwallet` daemons should work with default configuration for most users, however there is a wide variety of command line aguments to change the way they behave if required. For example, the following command can be used to change the log directory `dcrd` will write to.
+
+```bash
+dcrd --logdir=/my/custom/log/directory
+```
+
+`dcrd` and `dcrwallet` both support a `--help` argument which will print details of all the arguments they support and then exit immediately.
+
+---
+
+## `dcrd`
+
+??? info "Click to expand full list of `dcrd` arguments"
```

Comment:
Should be "Click to expand full list of `dcrd` CLI arguments" for consistency with the title and dcrctl's page. Same for line 100.